### DISCUSSION TITLE
Question about DBO

### DISCUSSION THREAD

**Discussion by JUNGWONN** [google/digitalbuildings#655] [Q&A]
Hi, I am Jungwon Lee,
 
Our company is also conducting business related to building ontology using brick schema. 
 
While searching for references, I got to know your DBO project and got a lot of inspiration. 
 
One question is that in order to calculate the energy optimization of the building, SPEC DATA such as heat capacity, voltage, and power of HVAC facilities are required. 
 
I wonder if there is a way to add information such as boiler capacity when configuring Ontology. 
If not,  I wonder if you have any plans later.
 
 
Best regards
 
Lee

**Comment by tasodorff** [comment]
Hi Jungwon,

The answer is yes: [these fields](https://github.com/google/digitalbuildings/blob/master/ontology/yaml/resources/fields/metadata_fields.yaml) defines a lot of fields meant for capacities and specifications. Note that we use certain subfields like `specification`, `capacity`, and `requirement` for fields intended to store metadata like that.

Hope that helps.

**Reply by tasodorff** [reply]
and, to extend the ontology to cover others, we would simply add fields that follow similar conventions.
